content/Day-15/inclass-activity/_index.md

```diff
@@ -1,6 +1,12 @@
 ---
 title: "Instructions"
-weight: 112
+weight: 115
 ---
 
-## Activity : Business Analytic 
\ No newline at end of file
+## Introduction to the capstone project
+This capstone is a summative assessment that should provide an opportunity for learners to synthesize large amounts of content and/or skills and to engage with course material creatively particularly in knowledge. In designing it thought carefully about the kind of mastery students should demonstrate. The purpose of this capstone project is for you to apply what you have learned throughout your time at Launch Code. We will be using cloud resources, documentation and technical details to build the architectural requirements. Below is a sample of what your finished product diagram for Microsoft Azure platform. You will be using Amazon as your cloud platform for this project. 
```

Comment:
I think this paragraph is also a little confusing because the first part and second parts sound like you are addressing two different groups of people.